site/source/_posts/2018/03/11/Dojo-2-rc1.md

```diff
@@ -0,0 +1,186 @@
+
+The [Custom Elements Everywhere](https://custom-elements-everywhere.com/) project was released last year to highlight framework support for custom elements, and Dojo currently has a perfect score. Within Dojo 2, you may use custom elements, express Dojo 2 widgets as custom elements, and easily export Dojo 2 widgets as custom elements via one of our `@dojo/cli` commands, making it easy to share and distribute custom elements for easy use across projects and frameworks! So if you're in an organization where some teams use Angular, some use React, some use Vue, and some use Dojo, we provide a future today where you can write a component once and share it across those teams!
+
+By default, Dojo's custom elements are built for evergreen browsers to reduce overall bundle size, meaning that most basic widgets are <20KB (gzipped) which includes all of the @dojo runtime needed to support the custom element.
+
+To create or use a Dojo 2 web component,
+
+```
+<div>
+	<demo-menu-item id="a" title="Menu Item A"></demo-menu-item>
+</div>
+```
+
+To compile a Dojo widget into a web component, run the following command:
+
+```
+npx @dojo/cli build widget --elements src/Helloworld
+```
+
+For ease of use, all [`@dojo/widgets`](http://github.com/dojo/widgets) are compiled to custom elements and published to npm for easy import into your applications.
+
+Learn more about creating your first Dojo widgets in the [first Dojo 2 application tutorial](https://dojo.io/tutorials/001_static_content/).
+
+## The Dojo CLI
+
+CLI tools provide an easy way to work with frameworks, a trend that was popularized in the JS community by Ember. The [Dojo CLI](http://github.com/dojo/cli) is a collection of packages, providing a CLI architecture which enables a modular approach to create and work with commands.
+
+One of our goals in creating an easy out of the box experience with Dojo is to abstract users away from complex configuration by providing intelligent defaults internally. These defaults are easily overridden by experienced users who prefer something beyond our recommended default options.
+
+To install the Dojo CLI, run the following command:
+
+`npm install -g @dojo/cli`
+
+The [`@dojo/cli-build-app`](http://github.com/dojo/cli-build-app) package provides powerful tools to code split dynamically imported widgets automatically. Support is planned for the near future to enable users to specify this purely by configuration.
+
+The CLI also offers Build Time Rendering. During build time, the CLI renders the application, extracts the resulting HTML into the application's index.html and inlines critical CSS. `@dojo/cli-build-app` is an initial minimum viable version with more refinements planned. Please let us know what feedback you may have!
```

Comment:
It's the build time rendering specifically that is MVP, not the command itself:

"The CLI also offers Build Time Rendering. During build time, the CLI renders the application, extracts the resulting HTML into the application's index.html and inlines critical CSS. `@dojo/cli-build-app` is an initial minimum viable version with more refinements planned. Please let us know what feedback you may have!" -> "cli-build-app` also offers basic support for Build Time Rendering. During an applications build, the command renders the application, extracts the resulting HTML into the index.html and inlines critical CSS. Build time rendering is an early minimal viable version with more refinements to come. Please let us know what feedback you may have!"